pkgs/native_assets_cli/test/model/build_output_test.dart

```diff
@@ -173,37 +137,99 @@ version: ${BuildOutputImpl.latestVersion}''';
   "version": "${BuildOutputImpl.latestVersion}"
 }''';
 
-  test('built info yaml', () {
-    final yaml = yamlEncode(buildOutput.toJson(BuildOutputImpl.latestVersion))
-        .replaceAll('\\', '/');
-    expect(yaml, yamlEncoding);
+  const jsonEncodingV1_0_0 = '''
+{
```

Comment:
This cannot be migrated to JSON. The old build.dart's will output YAML and we don't want to break that by accidentally replacing our YAML parsing with JSON and not catching it because our test doesn't exercise this.